src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+
+export const Layout: React.FC = () => {
+  return (
+    <div>
+      <Outlet />
+    </div>
+  );
```

Comment:
What is the point of this `Layout`? You can move your navigation and other layout elements here